### ISSUE TITLE
Simplify escaping without regex

### ISSUE THREAD

**Issue by amir-zeldes** [amir-zeldes/DepEdit#4]

**Comment by amir-zeldes** [comment]
Also need to be able to escape equals sign in action values such as:

```
lemma=/^(ⲙⲡⲣ|ⲟⲩⲕ|ⲟⲩⲇⲉ|ⲙⲙⲛ|ⲙⲙⲟⲛ)$/	none	#1:morph=Polarity=Neg
```

**Comment by amir-zeldes** [comment]
Fixed in f5f3956988266703dc129052478e270a8302c865
